Custom view docs changes (#3321)
commercetools/merchant-center-application-kit: website-custom-views/src/content/api-reference/custom-view-config.mdx

```json
{

>   "name": "The Avengers app"

Author: Reverted the commit: 46cee48b702614e01b30f84ec26a9533094936a3

## `locators`

> Since Custom Views are rendered in the context of built-in Merchant Center applications, you need to set in which specific views the Custom View should be rendered.

Reviewer: ```suggestion
Since Custom Views are rendered in the context of built-in Merchant Center applications, you need to define the specific views where the Custom View should be rendered.
```

## Environment variable references

> To make the Custom View config more reusable across different environments, you can use references to environment variables within the config file.

Reviewer: ```suggestion
To make the Custom View config reusable across different environments, you can use references to environment variables within the config file.
```
